### ISSUE TITLE
Side-effect import not sorted properly with @typescript-eslint 2.23.0

### ISSUE THREAD

**Issue by Livven** [lydell/eslint-plugin-simple-import-sort#39]
This is with `"eslint-plugin-simple-import-sort": "5.0.1"`.

Before (i.e. 2.22.0):

```js
import "hard-rejection/register";

import chalk from "chalk";
import dotenv from "dotenv";
import ora from "ora";
import pluralize from "pluralize";
```

After:

```js
import chalk from "chalk";
import dotenv from "dotenv";
import "hard-rejection/register";
import ora from "ora";
import pluralize from "pluralize";
```

[Here's the changelog](https://github.com/typescript-eslint/typescript-eslint/blob/master/CHANGELOG.md). Seems most likely to be caused by (completely uneducated guess) https://github.com/typescript-eslint/typescript-eslint/pull/1697?

**Comment by brandon-leapyear** [comment]
Possibly fixed with #35

**Comment by brandon-leapyear** [comment]
Seems like this broke it, by setting a default for `importKind`:

https://github.com/typescript-eslint/typescript-eslint/pull/1697/files#diff-68b2b0b0d2d0ff1846b1114472008332R1536

https://github.com/chimurai/eslint-plugin-simple-import-sort/blob/master/src/sort.js#L842

**Comment by lydell** [comment]
Fixed in 5.0.2. Thanks for the help!

**Comment by Livven** [comment]
Confirmed working for me, thanks for the quick fix!
